src/Mod/Path/PathScripts/PostUtilsParse.py

```diff
@@ -94,130 +94,144 @@ def drill_translate(values, outstring, cmd, params):
 
     # get the other parameters
     drill_feedrate = Units.Quantity(params["F"], FreeCAD.Units.Velocity)
-    if cmd == "G83":
+    if cmd in ("G73", "G83"):
         drill_Step = Units.Quantity(params["Q"], FreeCAD.Units.Length)
-        a_bit = (
-            drill_Step * 0.05
-        )  # NIST 3.5.16.4 G83 Cycle:  "current hole bottom, backed off a bit."
+        # NIST 3.5.16.4 G83 Cycle:  "current hole bottom, backed off a bit."
+        a_bit = drill_Step * 0.05
     elif cmd == "G82":
```

Comment:
Why elif and not else?

Reply:
That should have been an "if" rather than an "elif".  I will fix it.

Reply:
I am changing it back to an elif after thinking about it some more.  "cmd" could also be "G81" at this point which doesn't need to parse either Q or P parameters, so an "else" isn't correct and a plain "if" isn't quite as efficient, I think.